src/views/CompletionsOverview.vue

```diff
@@ -60,6 +60,13 @@ export default {
         params: { activeStreetFid: block, editmode: 'edit' },
       });
     },
+    formatDate(date) {
```

Comment:
Utils file for this function? up to you.

Reply:
We should do this but for now let's keep it as it is and factor out later